.buildkite/scripts/install_deps.sh

```diff
@@ -75,6 +79,15 @@ with_docker() {
 }
 
 with_docker_compose_plugin() {
+    if [[ "${platform_type_lowercase}" == "darwin" ]]; then
+        echo "Skip. Not supported in Darwin."
+        return
+    fi
+    if ! command -v docker &> /dev/null ; then
+        echo "Skip. Docker is not installed by default"
+        return
+    fi
```

Comment:
Or this.

Reply:
As mentioned here, I would keep this condition too.
In this case, if there is no `docker` there is no need to install docker-compose.